src/main/java/me/Danker/commands/SetAngleCommand.java

```diff
@@ -0,0 +1,40 @@
+package me.Danker.commands;
+
+import me.Danker.DankersSkyblockMod;
+import net.minecraft.client.Minecraft;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.util.ChatComponentText;
+
+
+public class SetAngleCommand extends CommandBase {
```

Comment:
Headsnapping is detectable by watchdog and would probably result in a ban.

Reply:
Is there a way to ask if such a feature is actually bannable?